lib/topological_inventory/providers/common/operations/source.rb

```diff
@@ -22,10 +25,13 @@ class Source
         attr_accessor :params, :request_context, :source_id, :account_number
 
         def initialize(params = {}, request_context = nil)
-          self.params          = params
-          self.request_context = request_context
-          self.source_id       = params['source_id']
-          self.account_number  = params['external_tenant']
+          self.params           = params
+          self.request_context  = request_context
+          self.source_id        = params['source_id']
+          self.account_number   = params['external_tenant']
+          self.messaging_host   = ENV['QUEUE_HOST']
+          self.messaging_port   = ENV['QUEUE_PORT']
+          self.checks_via_kafka = params['checks_via_kafka'] != false
```

Comment:
Where does this get set? Is it really needed if it's always going to be true?

I guess the bigger question is - I know this is here so we can have both, but is that really necessary? I think we should just cut straight to kafka-based updates.

Reply:
On second thought I see why this is configurable, just so we don't have to update all the specs everywhere. Rather than refactoring it out we should just put a ticket in the backlog for now so we can track that this refactor does need to be done eventually. :+1:

Reply:
@pkomanek I was more thinking about ENV variable than params value for this. 

@lindgrenj6 As this is one the most critical features I think it's useful to have a backup if something will be wrong with the kafka. 
At least I recommend to have the backup API solution on Prod for some time, refactoring can be done when kafka solution will be working without unexpected complications.

Reply:
Yes I agree, we should make it an ENV var that way we can flip it back if needbe.